src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -113,48 +132,140 @@ To chat with New Relic AI about log errors:
+    3. Look for the <DoNotTranslate>**Explain this error**</DoNotTranslate> button embedded in the **Stack trace panel**. New Relic AI will provide a summary of the error. 
 
-1. Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate> > <DoNotTranslate>**Errors inbox**</DoNotTranslate>.
-2. Select a workload.
-2. Look for the <DoNotTranslate>**Ask AI**</DoNotTranslate> button at the top of the log details panel. Our AI will provide a summary of the error. 
+You can dig deeper by asking follow-up questions. For example, you can say: 
 
-You can take your investigation further by telling our AI about your stack and asking follow up questions. For example, you can say: 
+```
+I have an Azure environment and am having issues with Redis being periodically unavailable. Can you tell me more about this error? How can I diagnose this issue? 
 
 ```
-Hi, I have an Azure environment and I am having issues with Redis being periodically unavailable. Can you tell me more about this error? How can I diagnose this issue? 
+## Fill alerting gaps [#coverage-gaps]
+
+As your team and systems evolve, staying ahead of potential issues is crucial. Alerts are your early warning system, but keeping them up to date can be challenging. New Relic AI can help identify areas where you might be missing critical notifications.
+
+Chat with New Relic AI and ask:
 
+```
+Show me all of my entities that are not covered by an alert. 
 ```
 
-## Secure data
+```
+Show me my OpenTelemetry services that are not covered by an alert.
+```
+
+## Create synthetic monitors [#synthetic]
 
-Data security and privacy are primary concerns for anyone building and maintaining software. We want you to know that anything you ask our AI is safe. NRAI is built with the same security controls and standards that safeguard New Relic customer data. We will never share your data with other accounts or customers.  We work with vendors that meet our privacy and security standards, and we carefully assess prospective vendors in these critical areas before engaging their services. 
+Synthetic monitoring helps you proactively catch and resolve issues before they affect your customers. You can now create a synthetic monitor by chatting with our AI. Using New Relic AI, create a ping check monitor from the default public location (Portland, Oregon) and check the availability of a URL at your desired cadence. 
 
-Our AI uses a variety of inputs to improve its responses and always provides the best answer. For example, it uses inputs like our product documentation, a large library of example queries, and your telemetry data. But even though our AI scans your data as needed to provide context and help you build charts and understand response times, that data is _never stored_ by New Relic.
+For example:
 
-## Always learning 
+```
+Create a synthetic monitor to check the URL 'newrelic.com' every 10 minutes and name it: 'my awesome monitor'.
```

Comment:
@homelessbirds how does this prompt ensure that the monitor is in Portland?

Reply:
It just defaults to Portland. There's no other option.